### PR TITLE
Section about vsphere credentials in cluster lifecycle design doc

### PR DESCRIPTION
**Pull request by g-gaston**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: designs/full-cluster-lifecycle-api.md [hunk 1]
```diff
@@ -91,13 +91,23 @@ The cluster reconciler won’t continue with the reconciliation process until `V
...
+
+However this is a feature we can easily build on top of this design. We would have multiple secrets and open our API to be able to specify one set of credentials or the other (something like a reference to a secret). This would preserve backwards compatibility with the design we propose: if no credentials are specified, the controller will default to the ones used by the management cluster.
 ### New ClusterStatus 
 
 The [current Cluster Status](https://github.com/aws/eks-anywhere/blob/v0.6.0/pkg/api/v1alpha1/cluster_types.go#L287)will be modified to include extra fields and conditions. These will be used by the reconciler methods to trigger the appropriate actions and to allow users to check progress of operations.
 
 If an error occurs controllers will set `spec.ClusterStatus.FailureReason` and `spec.ClusterStatus.FailureMessage`.
 
-
+g
```
Comment (jiayiwang7) [inline designs/full-cluster-lifecycle-api.md:110]: what is this "g" about?

Comment (g-gaston) [reply designs/full-cluster-lifecycle-api.md]: lmao thanks!

Comment (jiayiwang7) [reply designs/full-cluster-lifecycle-api.md]: name initial 👍

**Review by jiayiwang7 — APPROVED** [review]

**Pull request merged**